doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
 ...
 ```
 
-### Run graphite in Docker - quick start
-
-If you don't have a default docker machine created yet:
-
-    docker-machine create --driver=virtualbox default
-
-Start a docker machine:
-
-    docker-machine start
-
-Make sure it is running:
-
-    $ docker-machine status
-    Running
-
-Run the following command:
+### Run Graphite & Grafana in Docker - quick start
 
-    $ docker run -d --name graphite --restart=always hopsoft/graphite-statsd
+The following commands will download the latest version of `kamon/grafana_graphite` docker image that contains both Grafana and Graphite, and start them while mounting local directory `./docker-grafana-graphite-master/data` for metrics persistance:
 
-Get the "local" IP address of the container:
+    $ curl -SL https://github.com/kamon-io/docker-grafana-graphite/archive/master.tar.gz | tar -xzf -
+    $ make -C docker-grafana-graphite-master up
 
-    $ docker inspect graphite | grep IPAdd | grep -v Secon | cut -d '"' -f 4 | head -n 1
-    172.17.0.2
+Go to http://localhost to view Grafana dashboard that's already set up to use metrics from Graphite.
 
-Get IP address of the machine:
+### Add metrics to Grafana dashboard
 
-    $ docker-machine ip
-    192.168.99.100
+We recommend a baseline of following metrics to track in your MongooseIM instance.
```

Comment:
alt:
We recommend the following metrics as a baseline for tracking your MongooseIM instance.